Dockerfile

```diff
@@ -88,4 +84,6 @@ EXPOSE 7681
 
 ENV APP_PASSWORD=
 
-ENTRYPOINT /usr/bin/tini /bin/ttyd -- --credential umbrel:${APP_PASSWORD} bash
+ENV USERNAME=umbrel
```

Comment:
@AaronDewes I wonder whether we could get rid of hardcoding "umbrel" here. Do you know whether Umbrel could also pass us the username?

Reply:
It's not really hardcoded, just a default value. It would probably make more sense to let Umbrel pass that in and change the default to something generic (Like USERNAME & APP_PASSWORD could default to lnshell)